src/components/contact/contact-page-form/ContactPageForm.jsx

```diff
@@ -143,14 +143,14 @@ function ContactPageForm() {
           />
         </div>
         <div className="flex justify-center w-full">
-          <HubblrGradientBorderButtonBase
+          <GradientButton
             isSubmitButton
             theme="dark"
             widthClass="max-sm:w-full"
```

Comment:
kommt nicht von dir, aber widthClass gibt es nicht mehr in den props, kann also raus

Reply:
Done